src/lib.rs

```diff
@@ -136,7 +136,10 @@ pub use crate::statistics::*;
 mod statistics;
 
 #[cfg(feature = "core")]
+pub use crate::util::{ComplexFloating, RealFloating, RealNumber};
```

Comment:
```suggestion
pub use crate::util::{FloatingPoint, ComplexFloating, RealFloating, RealNumber};
```

I think the `FloatingPoint` trait was forgotten? Because having this would also be helpful.

Reply:
Good catch, might have missed while reorganizing them into multiple lines.